dags/glam.py

```diff
@@ -132,7 +132,7 @@
     sql_file_path=f"sql/{table_project_id}/{dataset_id}/clients_scalar_aggregates_v1/query.sql",
     project_id=billing_project_id,
     depends_on_past=True,
-    arguments=("--replace",),
+    arguments=("--replace", "--reservation_id=moz-fx-bigquery-reserv-global:glam"),
```

Comment:
Location also needs to be set
```suggestion
    arguments=("--replace", "--reservation_id=moz-fx-bigquery-reserv-global:US.glam"),
```